stacks/serverless/dovetail-cdn-arranger.yml

```diff
@@ -0,0 +1,133 @@
+# stacks/serverless/dovetail-cdn-arranger.yml
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Dovetail CDN Arranger lambda function
+Conditions:
+  IsProduction: !Equals [!Ref EnvironmentType, Production]
+Parameters:
+  CodeS3Bucket:
+    Type: String
+  CodeS3ObjectVersion:
+    Type: String
+  EnvironmentType:
+    Type: String
+  DovetailHost:
+    Type: AWS::SSM::Parameter::Value<String>
+  DovetailToken:
+    Type: AWS::SSM::Parameter::Value<String>
+  DynamodbTableArn:
+    Type: AWS::SSM::Parameter::Value<String>
+Resources:
+  DovetailCdnArrangerWorkspaceBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      # TODO: tag-specific expirations
+      LifecycleConfiguration:
+        Rules:
+          - ExpirationInDays: 30
+            Status: Enabled
+      Tags:
+        - Key: Project
+          Value: Dovetail
+        - Key: Environment
+          Value: !Ref EnvironmentType
+        - Key: prx:cloudformation:stack-name
+          Value: !Ref AWS::StackName
+        - Key: prx:cloudformation:stack-id
+          Value: !Ref AWS::StackId
+  DovetailCdnArrangerBinaryLayer:
+    Type: AWS::Lambda::LayerVersion
+    Properties:
+      CompatibleRuntimes:
+        - nodejs14.x
+        - nodejs12.x
+      Content:
+        S3Bucket: prx-tech
+        S3Key: archives/ffmpeg-4.3.1-lambda-layer.zip
+      Description: ffmpeg binaries for dovetail-cdn-arranger
+      LicenseInfo: MIT
+  DovetailCdnArrangerExecutionIAMRole:
+    Type: 'AWS::IAM::Role'
```

Comment:
Prefer no quotes here

Reply:
Bah- I think Atom+Prettier added all those.  Or at least changed them from double to single.  Will see if there's some other linting command that would remove.  Or i'll just remove them.